.gitignore

```diff
@@ -1,3 +0,0 @@
-spec/dummy
```

Comment:
I accidentally added .gitignore to my push. I deleted the whole file.

We gitignored the spec/dummy directory because it's used for testing.